collector/hwmon_linux.go

```diff
@@ -143,6 +144,26 @@ func (c *hwMonCollector) updateHwmon(ch chan<- prometheus.Metric, dir string) (e
 		}
 	}
 
+	hwmonChipName, err := c.hwmonHumanReadableChipName(dir)
+
```

Comment:
Please remove newline, it's uncommon to have a newline between returning and error and checking it.